packages/fx-core/src/component/error/deployError.ts

```diff
@@ -21,42 +22,67 @@ export class DeployExternalApiCallError extends ExternalApiCallError {
       statusCode ?? -1,
       undefined,
       undefined,
-      typeof error === "string" ? error : JSON.stringify(error)
+      typeof error === "string" ? error : JSON.stringify(error),
+      helpLink
     );
   }
 
-  static zipDeployError(e?: unknown, statusCode?: number): DeployExternalApiCallError {
+  static zipDeployError(
+    e?: unknown,
+    statusCode?: number,
+    helpLink?: string
+  ): DeployExternalApiCallError {
     return new DeployExternalApiCallError(
       DeployConstant.DEPLOY_ERROR_TYPE,
       "ZipDeployError",
       "plugins.bot.FailedDeployZipFile",
-      statusCode ?? -1
+      statusCode ?? -1,
+      undefined,
+      undefined,
+      undefined,
```

Comment:
There are too many undefined....

Reply:
It seems to be no good way to avoid this